adapters/adapter.go

```diff
@@ -3,6 +3,7 @@ package adapters
 import (
 	"context"
 	"crypto/tls"
+	"github.com/prebid/prebid-server/family"
```

Comment:
I would lean towards naming this package `constants`, to make it more reusable. Named `family`, nothing else will go in there.

There are a few other "magic string constants" which are shared throughout several other PBS modules though. For example, `BidderCode` and `ResponseStatus`. It'd be nice to have a place to move them all to, as time permits.